vhdbuilder/packer/vhd-scanning-exe-on-vm.sh

```diff
@@ -23,7 +23,13 @@ elif [[ "$OS_SKU" == "Ubuntu" ]] && [[ "$OS_VERSION" == "22.04" ]] && [[ "${ARCH
         echo "Azure CLI is not installed properly."
         exit 1
     fi
-elif [[ "$OS_SKU" == "Ubuntu" ]] && [[ "$OS_VERSION" == "22.04" ]] && [[ "${ARCHITECTURE,,}" != "arm64" ]]; then
```

Comment:
Is there a change necessary for AMD64 as well. Because those are also failing for me

Reply:
Oh never mind, I saw that you changed it :)